Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb

```diff
@@ -0,0 +1,1041 @@
+{
```

Comment:
<p>Instead write (fixed typos and styling):</p><p> </p><blockquote>The following notebook uses a breast cancer dataset from USI machine learning repository that focuses on whether after seeing certain features, a patient has what type of cancer, i-e malignant or benign. &nbsp;It is followed by a series of steps involving data reading the analyzing, applying Machine Learning Algorithm the SVM Classifier and then printing out the accuracy score of this classifier. </blockquote><blockquote>We use the <a href="https://archive.ics.uci.edu/ml/datasets/Breast+Cancer+Wisconsin+(Diagnostic)" target="_blank">Breast Cancer Dataset</a></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>